projects/packages/jitm/src/class-post-connection-jitm.php

```diff
@@ -391,7 +391,7 @@ public function get_messages( $message_path, $query, $full_jp_logo_exists ) {
 		$use_cache = false;
 
 		/** This filter is documented in class.jetpack.php */
```

Comment:
Maybe it would be a good occasion to bring the docblock to the package itself, closer to the relevant code?

```suggestion
		/**
		 * Filter to turn off jitm caching
		 *
		 * @since 5.4.0
		 *
		 * @param bool false Whether to cache just in time messages
		 */
```